File: site/code-snippets/WebRtc/deleteParticipant/sample.php
<?php

require "vendor/autoload.php";

$BW_USERNAME = getenv("BW_USERNAME");
$BW_PASSWORD = getenv("BW_PASSWORD");
$BW_ACCOUNT_ID = getenv("BW_ACCOUNT_ID");

$config = new BandwidthLib\Configuration(
    array(
        'webRtcBasicAuthUserName' => $BW_USERNAME,
        'webRtcBasicAuthPassword' => $BW_PASSWORD,
    )
);
$client = new BandwidthLib\BandwidthClient($config);

$webRtcClient = $client->getWebRtc()->getClient();

$participantId = "1234-abcd";

try {
    $webRtcClient->deleteParticipant($BW_ACCOUNT_ID, $participantId);
} catch (BandwidthLib\APIException $e) {
    print_r($e->getResponseCode());
}

File: site/code-snippets/WebRtc/getParticipantSubscriptions/sample.rb
require 'bandwidth'

include Bandwidth
include Bandwidth::WebRtc

BW_USERNAME = ENV['BW_USERNAME']
BW_PASSWORD = ENV['BW_PASSWORD']
BW_ACCOUNT_ID = ENV['BW_ACCOUNT_ID']

bandwidth_client = Bandwidth::Client.new(
    web_rtc_basic_auth_user_name: BW_USERNAME,
    web_rtc_basic_auth_password: BW_PASSWORD
)

web_rtc_client = bandwidth_client.web_rtc_client.client

participant_id = "1234-abcd"
session_id = "4321-dcba"

begin
    #response = web_rtc_client.get_participant_subscriptions(BW_ACCOUNT_ID, participant_id, session_id)
    response = web_rtc_client.get_participant_subscriptions(BW_ACCOUNT_ID, session_id, participant_id)
    #NOTE: This is currently improperly defined
    puts response.data
rescue APIException => e
    puts e.response_code
end